web/i18n/defaults/active.en.toml

```diff
@@ -23,8 +23,14 @@ AdminAllowListRemove = "Remove"
 AdminAllowListRemoveConfirmWelcome = "Are you sure you want to remove this member? They will lose their alias, if they have one."
 AdminAllowListRemoveConfirmTitle = "Confirm member removal"
 
+AdminInvitesTitle = "Invites"
+AdminInvitesWelcome = "Here ytou can create invite tokens for people who are not yet members of this room."
```

Comment:
```suggestion
 AdminInvitesWelcome = "Create invites for people who are not yet members of this room."
 ```
 
 (slight rewrite above; otherwise just fix the typoe `ytou` -> `you`)

Reply:
Question, I still haven't figured out how to make these commit suggestions. How can haz?

Reply:
:D

comment on a specific line or line range using:
![image](https://user-images.githubusercontent.com/3862362/109952536-3db8bb00-7cdf-11eb-9911-d45d20a4bddf.png)

Reply:
Totally not obvious. Thanks for the pro tip!